drivers/led/is31fl3194/is31fl3194.c

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (c) 2022 Benjamin Björnsson <benjamin.bjornsson@gmail.com>.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT issi_is31fl3194
+
+/**
+ * @file
+ * @brief LED driver for the IS31FL3194 I2C LED driver
+ */
+
+#include <zephyr/zephyr.h>
+#include <zephyr/drivers/led.h>
+#include <zephyr/drivers/i2c.h>
+#include <string.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(is31fl3194, CONFIG_LED_LOG_LEVEL);
+
+#include "is31fl3194_reg.h"
+
+#define IS31FL3194_INDEX_MAX 2
+
+struct is31fl3194_config {
+	struct is31fl3194_ctx ctx;
```

Comment:
I am not sure, but I think this could be `const` because the pointers inside are not meant to be written. Isn't it ?

Reply:
I don't think const is necessary inside `struct is31fl3194_config`. The entire struct will be declared as `const` when used in the device macro.

Reply:
OK